docs/concepts/what-is-an-activity-task-execution.md

```diff
@@ -7,6 +7,8 @@ tags:
   - explanation
 ---
 
-An Activity Task Execution is the execution of an Activity function.
+An Activity Task Execution is when the Worker uses the Context provided from the [Activity Task](/docs/concepts/what-is-an-activity-task) and executes the [Activity Definition](/docs/concepts/what-is-an-activity-definition).
 
-Activity Task Executions are retried per a Retry Policy.
+From the API, users see an Activity being executed and assume that it runs once.
+
+In reality, the Activity can be retried per a Retry Policy until it achieves its expected result.
```

Comment:
```suggestion
The API to schedule an Activity Execution provides an "effectively once" experience, even though there may be several Activity Task Executions that take place to successfully complete an Activity.
```

Reply:
Could add more about the Events associated with an Activity Task Execution - like I mention below.